lib/rules/no-wait-for-multiple-assertions.ts

```diff
@@ -62,14 +85,30 @@ export default createTestingLibraryRule<Options, MessageIds>({
 
 			const expectNodes = getExpectNodes(node.body);
 
-			if (expectNodes.length <= 1) {
-				return;
+			const expectArgumentMap = new Map<
+				string,
+				TSESTree.ExpressionStatement[]
+			>();
+
+			for (const expectNode of expectNodes) {
+				const argument = getExpectArgument(expectNode.expression);
+				if (!argument) {
+					continue;
+				}
+
+				const argumentText = getSourceCode(context).getText(argument);
+				const existingNodes = expectArgumentMap.get(argumentText) ?? [];
+				const newTargetNodes = [...existingNodes, expectNode];
+				expectArgumentMap.set(argumentText, newTargetNodes);
 			}
 
-			for (let i = 0; i < expectNodes.length; i++) {
-				if (i !== 0) {
+			for (const expressionStatements of expectArgumentMap.values()) {
+				if (expressionStatements.length <= 1) {
+					continue;
+				}
+				for (const expressionStatement of expressionStatements.slice(1)) {
```

Comment:
suggestion: please add a comment clarifying why we are slicing from index 1.

I know this is to report from the 2nd occurrence of the same target, but better to leave an explicit comment about it.

Reply:
Added a comment to clarify why we slice from index 1. Thanks for the suggestion! [908e84e](https://github.com/testing-library/eslint-plugin-testing-library/pull/1097/commits/908e84e5dfe0857e470160b4278e439030b887cb)